### ISSUE TITLE
string x.toLower

### ISSUE THREAD

**Issue by thijmenboot** [Di0n/LowerCaseAPI#1]
This API is now redundant.. we have been patched.

**Comment by Di0n** [comment]
You don't seem to understand. Some people don't want to use string.lower. Some people want to receive a lower case string from a server. It puts less stress on the client's machine that way.

**Comment by thijmenboot** [comment]
I don't think you are fully grasping this concept. I meant we send this string to our satellite network which then processes it using random solar flare bursts. This is then sent to our API centre and returned to the client PC's using a JSON call. You are correct about the less stress on the client's apparatus.

**Comment by Di0n** [comment]
This is what I think about your proposal

![56cdca7abec0a image](https://cloud.githubusercontent.com/assets/7166725/26543867/dfb7982a-445f-11e7-8b24-9c7dde9de858.jpg)

**Comment by thijmenboot** [comment]
I think this would be your best bet:

```
namespace Columbine
 {

List<Child> childrenList = new List<Child>(new child("Klebold", "Dylan");
int ammo = 10;

foreach(Child child in childrenList)
{
   child.Dispose();
   ammo--;
}
}
```
